VERSION

```diff
@@ -1 +1 @@
-3.6.2
+3.7.0
```

Comment:
I would actually recommend separating the version bump into a different commit; typical practice is to land features onto master and then after master passes tests, create an additional commit on top of master that bumps the version (and changes the `CHANGELOG` from "NEXT" to the actual version number).